addons/website/views/snippets/s_process_steps.xml

```diff
@@ -84,6 +85,17 @@
             </we-row>
         </div>
     </xpath>
+    <xpath expr="//t[@t-call='website.snippet_options_background_options']" position="after">
+        <t t-call="website.snippet_options_background_options">
+            <t t-set="background_label">Step Background</t>
+            <t t-set="selector" t-value="'.s_process_step'"/>
+            <t t-set="target" t-value="'.s_process_step_number'"/>
```

Comment:
I'm not sure anymore, sorry. Placing the background controller into the "Column" entry streamlines the UI a bit, but it's not the expected location for the widget 😕 

I propose this :
```suggestion
            <t t-set="selector" t-value="'.s_process_step .s_process_step_number'"/>
```